components/team/Styles.js

```diff
@@ -46,4 +46,19 @@ const MemberItem = styled.li`
   margin-bottom: 5px;
 `;
 
-export { Box, Title, Form, Section, Strong, MemberItem };
+const StyledButton = styled.button`
```

Comment:
bit odd to have this inside the team folder if its used by outside components but i will approve to not slow things down

Reply:
I know, I didn't wanna move it out and then have to check every view 😄

Reply:
But now it's done, happy? 😛

Reply:
So happy